.travis.yml

```diff
@@ -4,3 +4,6 @@ node_js:
 script:
 - npm run test
 - npm run lint
+deploy:
+  provider: azure_web_apps
+  on: develop
```

Comment:
Don't we have username and password?

Reply:
I setup a couple of environment variables in travis directly: https://travis-ci.org/Rostlab/JS18_ProjectB_Group3/settings